foia_hub/templates/contacts/profile.html

```diff
@@ -52,30 +52,27 @@
         </p>
       </div>
 
-      <section class="library">
-        <p>
-            Before you submit a FOIA request:
-
-            <ul>
-                {% if profile.foia_libraries %}
-                    <li>
-                    Browse the agency's
-                    {% for r in profile.foia_libraries %}
-                        <a class='foialibrary' href="{{r.url}}">{{r.link_text}}</a>,
```

Comment:
Just looked at the new design. Looks great :+1: ;however, removing `foialibrary`, `foiagov--search--link` and `agency--site` will cause the event analytics to break for these links.